app/models/miq_queue.rb

```diff
@@ -179,6 +179,12 @@ def self.put_deprecated(*args)
     put(*args)
   end
 
+  # Place holder for deprecated put_unless_exists call
+  # which might not be needed in the new architecture
+  def self.put_unless_exists_deprecated(*args)
```

Comment:
I don't understand this at all.  The entire purpose of deprecating something is to force developers to:

* not add new calls
* replace existing calls with the suggested replacement

If we need to remove these, deprecate it.  If not, don't even call it put_unless_exists_deprecated.